sprokit/src/sprokit/pipeline/CMakeLists.txt

```diff
@@ -112,6 +112,7 @@ target_link_libraries(sprokit_pipeline
                       vital_logger
                       ${Boost_CHRONO_LIBRARY}
                       ${Boost_SYSTEM_LIBRARY}
+                      sprokit_python_util
```

Comment:
Ack! This is doomed to failure; there is no such library. (Pedantically: there is no such library *yet*; the directory that creates it has not been processed when this is processed.) Also, it seems that this needs to be conditional on Python being enabled?

Reply:
Actually... maybe not *fail* as such (since obviously CMake doesn't just croak and die), but I'm not sure CMake will recognize this as a target, ever. (I'm not sure if link resolution happens in a separate pass.) But at any rate, I suspect it would be better to not try to link a library target that hasn't been defined yet.

Reply:
It definitely builds and runs as is. It could be extraneous, I'll build it without the target later and see what happens. Off the top of my head, I can probably replace it with ${PYBIND11_LIBRARY}, which will be either empty if python is disabled or filled as necessary.

Reply:
#380 should take care of this...

Reply:
@hughed2, when the library is unknown, CMake generates a link command for an unknown library, i.e. `-lsprokit_python_util`. This *may* work by accident, but I'm not sure that dependencies are tracked correctly.

It definitely does *not* work if Python is not enabled, because in that case the `sprokit_python_util` library does not exist. (Unless from a stale build...)